lib/components/Provider/index.tsx

```diff
@@ -143,10 +146,11 @@ export const NotificationAPIProvider: React.FunctionComponent<
     });
 
     //  identify user
-    client.identify({
-      email: config.user.email,
-      number: config.user.number
-    });
+    if (config.connectionMode === 'live')
```

Comment:
Please research other tools to see if it's possible to do this in a nicer way, so we don't have to use if/else for every network call.